docs/mindsdb-docs/docs/sql/api/select.md

```diff
@@ -2,47 +2,93 @@
 
 ## Description
 
-The `#!sql SELECT` statement is used to get a predictions from the model table. The data is not persistent and is returned on the fly as a result-set. The basic syntax for selecting from the model is:
+The `#!sql SELECT` statement fetches predictions from the model table. The data is returned on the fly and not saved.
+
+But there are ways to save predictions data! You can save your predictions as a view using the [`CREATE VIEW`](/sql/create/view/) statement. Another way is to insert your predictions into a table using the [`INSERT INTO`](/sql/api/insert/) statement.
```

Comment:
`You can save your predictions as a view` - that is not saving of prediction. In `view` user may save query to make predictions.

Reply:
You are correct.
I have mentioned that on the `CREATE VIEW` page (PR #2909 is already merged) as follows:
> The `#!sql CREATE VIEW` statement creates a view, which is a saved `SELECT` statement executed every time we call this view, as opposed to a table that stores its data in columns and rows.

I will make the correction here.

Reply:
There is a link to the `CREATE VIEW` page, and also I made this correction:
> But there are ways to save predictions data! You can save your predictions as a view using the [`CREATE VIEW`](/sql/create/view/) statement. Please note that a view is a saved query and does not store data like a table. Another way is to insert your predictions into a table using the [`INSERT INTO`](/sql/api/insert/) statement.